pnpm-lock.yaml

```diff
@@ -2412,56 +2412,67 @@ packages:
     resolution: {integrity: sha512-u72Mzc6jyJwKjJbZZcIYmd9bumJu7KNmHYdue43vT1rXPm2rITwmPWF0mmPzLm9/vJWxIRbao/jrQmxTO0Sm9w==}
     cpu: [arm]
     os: [linux]
+    libc: [glibc]
```

Comment:
Not sure why this got added but I just did `git revert --no-commit @` followed by `pnpm i` so I'm leaving it in.